python/nano/notebooks/pytorch/requirements.txt

```diff
@@ -0,0 +1,9 @@
+pytest==7.1.2
+nbmake==1.3.0
+torch_ipex @ https://intel-optimized-pytorch.s3.cn-north-1.amazonaws.com.cn/wheels/v1.9/torch_ipex-1.9.0-cp37-cp37m-linux_x86_64.whl
+torchvision @ https://download.pytorch.org/whl/cpu/torchvision-0.10.0%2Bcpu-cp37-cp37m-linux_x86_64.whl
+lightning-bolts==0.5.0
```

Comment:
This file is only for the cifar10 example, not all future examples right?

Reply:
Done